web/app/components/new/doc-form.ts

```diff
@@ -116,7 +99,12 @@ export default class NewDocFormComponent extends Component<NewDocFormComponentSi
    * Validates the form and updates the `formErrors` property.
    */
   private validate() {
-    this.formErrors = { ...FORM_ERRORS };
```

Comment:
You can see here that we were repeatedly assigning an empty object to the `formErrors` property instead of having it reflect the actual state of the form.